src/Common/Exception/Unsupported.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * SocialConnect project
+ * @author: Patsura Dmitry https://github.com/ovr <talk@dmtry.me>
+ */
+
+namespace SocialConnect\Common\Exception;
+
+class Unsupported extends \SocialConnect\Common\Exception
+{
+    public function __construct($message = 'Unsupported functionality', $code = 0, \Exception $previous = null)
```

Comment:
The last argument should be typehinted as `Throwable` instead of `Exception`.

Reply:
@ADmad Can you explain why?) Thank you!

Reply:
In PHP 7 `Throwable` is the base interface for all exceptions. As you can see [here](https://www.php.net/manual/en/class.exception.php) the type for `$previous` argument is `Throwable` not `Exception`.